src/overview.md

```diff
@@ -29,7 +29,7 @@ The [`Rng`] trait provides a layer of convenience on top of [`RngCore`], whose
 highlights are:
 
 -   [`Rng::gen()`] provides a random value of any type supporting the [`Standard`] distribution.
--   [`Rng::gen_range(low, high)`] provides a uniform random value within the given range.
+-   [`Rng::gen_range(low..high)`] provides a uniform random value within the given range.
```

Comment:
Thanks. Can we clarify that this is half-open (includes only `low`) and add a second row for `low..=high` here?

Reply:
I've added a `low..=high` example in 1f84098 and explained the differences. I also noticed that the `overview.md` file isn't linked anywhere, probably for a separate PR but should we link it in SUMMARY.md?

Reply:
Aha, that's probably why this file didn't get updated. I don't remember exactly how this ended up not getting linked but it was in #10; possibly the plan was to delete this file.

What do you think we should do with it?

Reply:
Apologies for the late reply!

`overview.md` looks useful, giving a general overview of the API. I initially stumbled upon the wrong documentation when reading the `guide-values` page online though. I only fixed `overview.md` because doing a search of the method in this repo yielded more outdated examples.

I do believe that reintroducing `overview.md` (if you consider it worth it) will require some restructuring of the other content and page navigation. I'd say that's a separate piece of work from fixing the documentation examples? If we merge this PR the examples will be correct for people looking up this method (which might be beginners to the Rust language as that's how I stumbled upon this through one of the beginner tutorials). What we decide to do with the overview file can still be handled in a separate PR?

Reply:
That sounds okay to me. I'll go ahead and merge this then.

Are you offering to do some restructuring too? The book would likely benefit being edited from multiple points of view. I'll try to answer questions as they come up.